iroh-blobs/src/store.rs

```diff
@@ -66,7 +66,9 @@ impl TempCounterMap {
 
     fn dec(&mut self, value: &HashAndFormat) {
         let HashAndFormat { hash, format } = value;
-        let counters = self.0.get_mut(hash).unwrap();
+        let Some(counters) = self.0.get_mut(hash) else {
+            return;
```

Comment:
when would this happen?

Reply:
If you decrement something that has not been incremented before. It should never happen, but if it happens it's better to just ignore it than to panic. Given that messages that trigger a dec go via the rpc boundary I don't think it is safe to write unreachable here.

Maybe a warning?

Reply:
given that we don't expect this happening, putting in a warning would be good, so we know if it starts happening